eng/pipelines/templates/steps/cosmos-emulator.yml

```diff
@@ -3,14 +3,14 @@ parameters:
   StartParameters: ''
 
 steps:
-  - powershell: |
+  - pwsh: |
```

Comment:
Please verify that this doesn't regress https://github.com/Azure/azure-sdk-for-java/pull/5633. I believe we will have to do the Import-Module in `powershell` if we later want to call those modules for the certificate work.

Reply:
Right now it doesn't regress anything. This template is only used in python. I tested the pipeline [here](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=182686&view=results). I guess if the imported modules are later used for certificate work them it might be an issue. We could just leave it as powershell but right now switching to pwsh doesn't break anything.

Reply:
Given we plan to use this in the other languages and we know it is an issue in java I'd like to just keep it as "powershell" until we can confirm it doesn't break that build.